utils/get-shared-games.ts

```diff
@@ -0,0 +1,29 @@
+"use server";
```

Comment:
Instead of fetching this on client using server action I would fetch all data in server component (page)